bitnami/influxdb/templates/influxdb/configmap-initdb-scripts.yaml

```diff
@@ -3,7 +3,8 @@ apiVersion: v1
 kind: ConfigMap
 metadata:
   name: {{ include "influxdb.fullname" . }}-initdb-scripts
-  labels: {{- include "influxdb.labels" . | nindent 4 }}
+  labels: 
+    {{- include "influxdb.labels" . | nindent 4 }}
```

Comment:
In think in the template its much more readable as its on the exact same space as the labels created from it.

Reply:
Honestly, I don't have a strong opinion about this (probably because I'm very used to this way to declare them and I consider it less readable)
I'd rather keep it as it's now but, as I said, no strong opinion. I'm not blocking the PR for this